taker-frontend/src/App.tsx

```diff
@@ -169,68 +175,92 @@ export const App = () => {
                 nextFundingEvent={nextFundingEvent}
                 referencePrice={referencePrice}
             />
-            <Box textAlign="center" padding={3} bg={useColorModeValue("gray.50", "gray.800")}>
-                <Center marginTop={20}>
-                    <Routes>
-                        <Route path="/">
-                            <Route
-                                path="wallet"
-                                element={
-                                    <>
-                                        <Wallet walletInfo={walletInfo} />
-                                    </>
-                                }
-                            />
-                            <Route
-                                element={
-                                    // @ts-ignore: ts-lint thinks that {children} is missing but react router is taking care of this for us
-
-
-                                        <PageLayout
-                                            cfds={cfds}
-                                            connectedToMaker={connectedToMaker}
-                                            showPromoBanner={isWithinPromoPeriod}
-                                        />
-
-                                }
+            <Box
+                bg={useColorModeValue("white", "black")}
+                bgGradient={useColorModeValue(
+                    "linear(to-r, white 20%, gray.800, white 80%)",
+                    "linear(to-r, gray.800 20%, white, gray.800 80%)",
+                )}
+            >
+                <Center>
+                    <Box
+                        maxWidth={(VIEWPORT_WIDTH + 400) + "px"}
+                        width={"100%"}
+                        height={"100%"}
+                        bg={useColorModeValue("gray.200", "gray.700")}
+                    >
```

Comment:
I almost love it 😬 Imho we have too many grays. The outer one clashes with the navbar which is always visible.

![image](https://user-images.githubusercontent.com/224613/166195667-377e2c78-117f-4a18-a9d3-b8277432a540.png)

Reply:
oh, and I realized I don't have the fancy gradient ...

Reply:
maybe it's time to introduce the ItchySats logo's golden colour? :) or the ItchySats wallpaper in the back? ;D

Reply:
> I almost love it 😬 Imho we have too many grays.

Totally agree! I saw it more as a placeholder until we have something "designed" (e.g. trading CSS pattern, logo, ...) that we can put there.
Would you say this has to be fixed before we merge?

> oh, and I realized I don't have the fancy gradient ...

The gradient's problem is the way I use it because I define only one gradient and use a percentage distribution over the whole screen. That won't scale. Can be fixed but will need a bit of time to get it right. Maybe it's better to remove it for now.

> maybe it's time to introduce the ItchySats logo's golden colour? 

colors: I think we should invest some time into our color palette, but I am just not really good at it 😅

Reply:
>  Would you say this has to be fixed before we merge?

No, because I wouldn't know how to make it better.

Reply:
> Totally agree! I saw it more as a placeholder until we have something "designed" (e.g. trading CSS pattern, logo, ...) that we can put there.

Run ads? :D

Reply:
I removed the area and changed the gradient to feel more like a shadow:

![image](https://user-images.githubusercontent.com/5557790/166427039-d06e682f-d7ae-442d-8bed-b4780f3e44ec.png)

I tried to make the gradient of fixed size, but it's hard to achieve this in the current arrangement, because I cannot easily create a horizontal layout with fixed side elements where the center element is always stretched to the maximum view-port width. 
So, when making the window smaller the gradient eventually disappears at a certain window size. I think this is acceptable for now 🤷‍♂️